paper/paper.md

```diff
@@ -0,0 +1,88 @@
+<!--
+SPDX-FileCopyrightText: ASSUME Developers
+
+SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+---
+title: 'ASSUME - Agent-based Simulation for Studying and Understanding Market Evolution'
+tags:
+  - Python
+  - agent based simulation
+  - energy market
+  - reinforcement learning
+  - market simulation
+  - simulation
+authors:
+  - name: Florian Maurer
+    orcid: 0000-0001-8345-3889
+    corresponding: true
+    affiliation: 1
+    "affiliation": 1
+  - name: Nick Harder
+    orcid: 0000-0003-1897-3671
+    "affiliation": 2
+  - name: Kim K. Miskiw
+    orcid: 0009-0009-1389-4844
+    affiliation: 3
+  - name: Manish Khanra
+    orcid: 0000-0002-3347-9922
+    affiliation: 4
+
+affiliations:
+ - name: University of Applied Sciences Aachen, Germany
+   index: 1
+ - name: Inatech Universität Freiburg, Germany
```

Comment:
Institute for Sustainable Systems Engineering, University of Freiburg, Freiburg, Germany